src/status_im/ui/screens/chat/styles/message/message.cljs

```diff
@@ -74,10 +74,7 @@
          (last-message-padding message)))
 
 (defn timestamp-content-wrapper [outgoing message-type]
-  {:flex-direction (if outgoing :row-reverse :row)
-   :width          (if (= :system-message message-type)
-                     300
-                     230)})
+  {:flex-direction (if outgoing :row-reverse :row)})
```

Comment:
@yenda is that all, lol? I trust you it's good 🙏

Reply:
yes the PR fixing all other kind of issue reintroduced that property which was deleted for large bubble PR https://github.com/status-im/status-react/pull/7393/files